config.ts

```diff
@@ -28,6 +28,11 @@ config.android = {
   icon_asset_background_path: "./app_data/assets/android/icon-background.png",
 };
 
+config.ios = {
+  app_id: "international.idems.plh-teens-tz",
```

Comment:
I think in other configs we have this as `plh_teens_tz` and you added some converter script that replaces hyphens by underscores

Reply:
Yeh, any underscores will be replaced by hyphens by our own scripts before compilation. But I think maybe it's preferable to use hyphens explicitly so that this ID matches how it appears in the App Store console?